geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DestroyGatewayReceiverCommand.java

```diff
@@ -61,32 +59,19 @@ public Result destroyGatewayReceiver(
     Set<DistributedMember> members = getMembers(onGroups, onMember);
 
     List<CliFunctionResult> functionResults =
-        executeAndGetFunctionResult(DestroyGatewayReceiverFunction.INSTANCE, ifExists, members);
+        executeAndGetFunctionResult(DestroyGatewayReceiverFunction.INSTANCE, null, members);
 
-    CommandResult result = ResultBuilder.buildResult(functionResults);
+    ResultModel result = ResultModel.createMemberStatusResult(functionResults, ifExists);
+    result.setConfigObject(new CacheConfig.GatewayReceiver());
```

Comment:
May be you could pass `null` as it is not required to be used in updateClusterConfig?

Reply:
Thanks. I missed this change on my first refactoring iteration.